### PR TITLE
submit numa-aware plugin

### PR DESCRIPTION
**Pull request by huone1**

### CODE REVIEW

**Comment by Thor-wl** [comment]
Please rebase the commits to one

File: pkg/scheduler/plugins/numaaware/numaaware.go [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+			}
+		}
+
+		if _, ok := pp.assignRes[task.UID]; !ok {
+			pp.assignRes[task.UID] = make(map[string]api.ResNumaSets)
+		}
+
+		pp.assignRes[task.UID][node.Name] = allResAssignMap
+
+		klog.V(3).Infof(" task %s's on node<%s> resAssignMap: %v",
```
Comment (wpeng102) [inline pkg/scheduler/plugins/numaaware/numaaware.go:150]: suggest use V(4) log level

File: pkg/scheduler/plugins/numaaware/numaaware.go [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+		allResAssignMap := make(map[string]cpuset.CPUSet)
+		for _, container := range task.Pod.Spec.Containers {
+			providersHints := policy.AccumulateProvidersHints(&container, node.NumaSchedulerInfo, resNumaSets, pp.hintProviders)
+			hit, admit := taskPolicy.Predicate(providersHints)
+			if admit != true {
+				return fmt.Errorf("plugin %s predicates failed for task %s container %s on node %s",
+					pp.Name(), task.Name, container.Name, node.Name)
+			}
+
+			klog.V(3).Infof("[numaaware] hits for task %s container '%v': %v on node %s, besthit: %v",
```
Comment (wpeng102) [inline pkg/scheduler/plugins/numaaware/numaaware.go:135]: suggest use V(4) log level

File: pkg/scheduler/plugins/numaaware/numaaware.go [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+	s := cpus.ToSlice()
+
+	for _, cpuID := range s {
+		mask.Add(cpuDetails[cpuID].NUMANodeID)
+	}
+
+	return mask.Count()
+}
+
+func normalizeScore(maxPriority int64, reverse bool, scores map[string]int64) {
```
[the remark this answers is not in the document]
Comment (huone1) [reply pkg/scheduler/plugins/numaaware/numaaware.go]: done

**Comment by huone1** [comment]
Close it and create new PR #1493 , because it is difficult to rebase it due to sync master commits
the review comment is fixed in PR#1493

**Pull request closed without merging**